.zshrc

```diff
@@ -0,0 +1 @@
+export PHOENIX_URI="localhost:4000"
```

Comment:
are you using nix? this value is defined in `.env_dev` so you can source that directly instead if you don't use nix